istio.deps

```diff
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "74393cf764c167b545f32eb895314e624186e5b6"
+    "lastStableSHA": "b6e2543232ea486685292a07f8ec68775c0f6b7d"
```

Comment:
Ideally we split this out if possible before merging. Maybe even the g-c-p part

Reply:
sure, I can split it out in separate PRs (including the go-control-plane part). There seems some test failures due to the Envoy update though. Could you review other part in the PR, we can hold the merge until the other PRs are merged. thanks.

Reply:
added the hold label.